src/screens/permission/views/grant-section/twin-role-table.tsx

```diff
@@ -0,0 +1,123 @@
+import { GuidWithCopy } from "@/shared/ui/guid";
+import { UserResourceLink } from "@/entities/user";
+import { PagedResponse } from "@/shared/api";
+import { formatToTwinfaceDate } from "@/shared/libs";
+import { Experimental_CrudDataTable } from "@/widgets";
+import { ColumnDef, PaginationState } from "@tanstack/react-table";
+import { toast } from "sonner";
+import {
+  TwinClass_DETAILED,
+  TwinClassResourceLink,
+} from "@/entities/twinClass";
+import {
+  PermissionGrantTwinRoles,
+  PermissionGrantTwinRoles_DETAILED,
+  usePermissionGrantTwinRolesSearchV1,
+} from "@/entities/twinRole";
+import { PermissionSchemaResourceLink } from "@/entities/permissionSchema";
+
+const colDefs: Record<
+  keyof Pick<
+    PermissionGrantTwinRoles,
+    | "id"
+    | "permissionSchemaId"
+    | "twinClassId"
+    | "twinRole"
+    | "grantedByUserId"
+    | "grantedAt"
+  >,
+  ColumnDef<PermissionGrantTwinRoles>
+> = {
+  id: {
+    id: "id",
+    accessorKey: "id",
+    header: "Id",
+    cell: (data) => <GuidWithCopy value={data.getValue<string>()} />,
+  },
+  permissionSchemaId: {
+    id: "permissionSchemaId",
+    accessorKey: "permissionSchemaId",
+    header: "Persmission Schema",
+    cell: ({ row: { original } }) =>
+      original.permissionSchema && (
+        <div className="max-w-48 inline-flex">
+          <PermissionSchemaResourceLink data={original.permissionSchema} />
+        </div>
+      ),
+  },
+  twinClassId: {
+    id: "twinClassId",
+    accessorKey: "twinClassId",
+    header: "Class",
+    cell: ({ row: { original } }) =>
+      original.twinClass && (
+        <div className="max-w-48 inline-flex">
+          <TwinClassResourceLink
+            data={original.twinClass as TwinClass_DETAILED}
```

Comment:
ℹ️ Right now this `as TwinClass_DETAILED` is needed to compile successfully 

However it will be redundant and should be remove after `hydrate` method is applied correctly. Lets have a call an discuss it